src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+@import './variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  margin: 0 auto;
+  gap: $gap;
+  padding: $padding;
+}
+
+.calendar__day {
+  display: flex;
+  position: relative;
+  height: $day-size;
+  width: $day-size;
+  box-sizing: border-box;
+  background-color: $background-color;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  border: solid $border-size black;
+  justify-content: center;
+  align-items: center;
+  transition:
+    background-color $animation-duration,
+    transform $animation-duration;
```

Comment:
The transition property can be shortened by combining the properties into one line, as they share the same duration and easing function. For example: `transition: background-color $animation-duration, transform $animation-duration;`.